packages/editor/src/components/post-schedule/style.scss

```diff
@@ -0,0 +1,21 @@
+.editor-post-schedule__panel {
+	width: 100%;
+	justify-content: flex-start;
+	align-items: flex-start;
+
+	span {
+		display: block;
+		width: 30%;
+	}
+}
+
+.editor-post-schedule__panel-dropdown {
+	width: 70%;
+}
```

Comment:
Some of the styles above are duplicated between these "panels". Since this is the second panel I'm adding, I think it's fine, but as a direct follow-up for this PR, I'll try to see if we can use a common UI component and share it across the panels.

Both post and site editor approaches are not satisfying for me:

 - Site editor reuses classnames, we should always reuse components and not classnames
 - Post editor duplicates styles.

Reply:
The follow-up is ready here https://github.com/WordPress/gutenberg/pull/56238